includes/class-woothemes-sensei-course.php

```diff
@@ -2020,4 +2023,25 @@ public static function the_course_action_buttons( $course ){
 
     }// end the_course_action_buttons
 
+    /**
+     * sensei_course_category_order_main_content function
+     *
+     * Sort the main query on custom taxonomy pages to pass posts in
+     * 'menu_order', because Sensei allows users to set the order of
+     * Courses under Course > Course Order.
+     *
+     * @since 1.9.0
+     * @access public
+     * @param object $query
+     * @return object $query
+     */
+
+    public function sensei_course_category_order_main_content($query) {
+        if ( is_tax() ) {
```

Comment:
@mahangu this will fire and modify the query for all taxonomies loaded on any given WP site. We'll need to be more restrictive as to which taxonomy we're targeting: is_tax( $taxonomy ) . If we're checking all of Sensei taxonomies we'll have to check fo each of them in the if statement.